### PR TITLE
v1.0.4_SSR-1399 Transfer blog (Releases) to docs template

### PR DESCRIPTION
**Pull request by nmanhit**

### CODE REVIEW

File: docs/document/docs/releases/release-notes-v1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+id: release-notes-v1.0.0
+title: v1.0.0 Release Notes
+sidebar_label: v1.0.0 Release Notes
+original_id: release-notes-v1.0.0
```
Comment (TomokoMiyake) [inline docs/document/docs/releases/release-notes-v1.0.0.md:5]: I think it is no need to specify `original_id` under docs folder.
This is for versioning.

File: docs/document/docs/releases/release-notes-v1.0.1.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+id: release-notes-v1.0.1
+title: v1.0.1 Release Notes
+sidebar_label: v1.0.1 Release Notes
+original_id: release-notes-v1.0.1
```
Comment (TomokoMiyake) [inline docs/document/docs/releases/release-notes-v1.0.1.md:5]: I think it is no need to specify `original_id` under docs folder.
This is for versioning.

File: docs/document/docs/releases/release-notes-v1.0.2.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+id: release-notes-v1.0.2
+title: v1.0.2 Release Notes
+sidebar_label: v1.0.2 Release Notes
+original_id: release-notes-v1.0.2
```
Comment (TomokoMiyake) [inline docs/document/docs/releases/release-notes-v1.0.2.md:5]: I think it is no need to specify `original_id` under docs folder.
This is for versioning.

File: docs/document/docs/releases/release-notes-v1.0.3.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+id: release-notes-v1.0.3
+title: v1.0.3 Release Notes
+sidebar_label: v1.0.3 Release Notes
+original_id: release-notes-v1.0.3
```
Comment (TomokoMiyake) [inline docs/document/docs/releases/release-notes-v1.0.3.md:5]: I think it is no need to specify `original_id` under docs folder.
This is for versioning.

File: docs/document/docs/releases/release-notes.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+id: release-notes
+title: v1.0.4 Release Notes
+sidebar_label: v1.0.4 Release Notes
+original_id: release-notes-v1.0.4
```
Comment (TomokoMiyake) [inline docs/document/docs/releases/release-notes.md:5]: I think it is no need to specify `original_id` under docs folder.
This is for versioning.

File: docs/document/docs/releases/release-notes-v1.0.0.md (hunk 1 above)
Comment (nmanhit) [reply docs/document/docs/releases/release-notes-v1.0.0.md:5]: Ok, i removed it

File: docs/document/docs/releases/release-notes-v1.0.1.md (hunk 2 above)
Comment (nmanhit) [reply docs/document/docs/releases/release-notes-v1.0.1.md:5]: Ok, i removed it

File: docs/document/docs/releases/release-notes-v1.0.2.md (hunk 3 above)
Comment (nmanhit) [reply docs/document/docs/releases/release-notes-v1.0.2.md:5]: Ok, i removed it

File: docs/document/docs/releases/release-notes-v1.0.3.md (hunk 4 above)
Comment (nmanhit) [reply docs/document/docs/releases/release-notes-v1.0.3.md:5]: Ok, i removed it

File: docs/document/docs/releases/release-notes.md (hunk 5 above)
Comment (nmanhit) [reply docs/document/docs/releases/release-notes.md:5]: Ok, i removed it

**Pull request merged**